gateway/api/searches/helpers.go

```diff
@@ -0,0 +1,75 @@
+package searches
+
+import (
+	"strings"
+	"time"
+
+	"github.com/go-git/go-git/v5/plumbing/object"
+	"github.com/hoophq/hoop/common/runbooks"
+)
+
+const RUNBOOK_CACHE_TTL = 1 * time.Minute
```

Comment:
I would increase this to 5~10 minutes. I don't see see too much importance on having this data always fresh for runbooks. Fresh data it's more important for administrators testing a runbook to make them available on Hoop.

Not a merge blocker, though